### PR TITLE
Add nice!nano development board

### PR DESCRIPTION
**Pull request by Nicell**
I've added the configuration for my nRF52840 based development board I've created. It will be available soon. There's some bare bones documentation available here that will get extended soon: https://docs.nicekeyboards.com/#/nice!nano/

Thanks for the work that's been put into this bootloader! It's been a joy to use.

### CODE REVIEW

File: src/boards/nice_nano/board.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// BLE OTA
+//--------------------------------------------------------------------+
+#define BLEDIS_MANUFACTURER  "Nice Keyboards"
+#define BLEDIS_MODEL         "nice!nano"
+
+//--------------------------------------------------------------------+
+// USB
+//--------------------------------------------------------------------+
+#define USB_DESC_VID           0x239A
+#define USB_DESC_UF2_PID       0x0029
```
Comment (hathach) [inline src/boards/nice_nano/board.h:55]: the VID/PID is already used and we don't allow to reuse it anymore. If your board can also run circuitpython, you could apply for a set of VID/PID from Adafruit.

Comment (Nicell) [reply src/boards/nice_nano/board.h:55]: I'm not familiar with Circuit Python. Is there any reason I wouldn't be able to use Circuit Python on this nRF52840 board? If not, could you direct me where to add support and apply for a VID/PID from Adafruit? Thanks!

Comment (hathach) [reply src/boards/nice_nano/board.h:55]: join circuitpyhon discords channel and ask for a donation of VID/PID to see if you are eligible for one.

**Comment by Nicell** [comment]
@hathach I've added my unique PID. See: https://github.com/adafruit/circuitpython/issues/3013

**Pull request merged**